crates/swc_core/Cargo.toml

```diff
@@ -30,11 +41,29 @@ perf = ["swc_common/perf"]
 # refer binding_core_wasm for example: it has plugin_transform_host feature, but
 # it has to be a feature to be enabled by compile time flag to avoid cargo's feature collision,
 # while it also need to access 'common' exports by default for cargo check / or clippy.
-common = ["swc_common"]
+common = ["__common"]
+# TODO: internally concurrent / perf refers to the same feature (`parking_lot`)
+# - can we consolidate them?
+common_concurrent = ["__common", "swc_common/concurrent"]
```

Comment:
q: what's difference between common/perf vs. common/concurrent? it looks like `/perf` enables specific for some hashmap, curious if `concurrent` can includes everything or it have dinstict purpose.

Reply:
`concurrent`allows using `Lrc` and some types in multi-threaded context.
(i.e. They implements `Sync` and `Send`